addons/headless/functions/fnc_handleSpawn.sqf

```diff
@@ -22,7 +22,7 @@ TRACE_1("Spawn",_object);
 if (!(_object in allUnits) || {isPlayer _object}) exitWith {};
```

Comment:
Comment above this line:
> // Exit if HC transferring disabled or object not a unit (including unit inside vehicle) or is player

I tested it and `allUnits` returns units that are in vehicles. Have I misunderstood the comment or is the comment wrong? Does the code need to be updated to match the comment?

Reply:
This is probably here because of `AllVehicle` above. But maybe there can be things in `CAManBase` that doesn't appear in `allUnits`?

Reply:
Without a doubt.

However, with the review, I was referring to the content of the comment:

> // Exit if HC transferring disabled or object not a unit (including unit inside vehicle) or is player

`(including unit inside vehicle)` perturbs me, as it does not apply here. I'm wondering if I should update the code to match the comment or the comment to match the code.

Reply:
The first part of the comment is wrong, because the "transferring disabled" check got moved to `settingsInitialized` [here](https://github.com/acemod/ACEX/commit/d3197ab741abcdc171d2d17b10a89617c1654d88#diff-9bd2a26504464499ffeb944092e04597bf2cbb4060dfb312cadbca2e21152b79).

As for `(including unit inside vehicle)` I looked at the entire history. All this comment means to say is that `allUnits` includes units inside vehicles and those are checked too, which is correct with what the code does.